wire-ios-sync-engine/Source/UserSession/ZMUserSession+SecurityClassification.swift

```diff
@@ -26,9 +26,14 @@ public enum SecurityClassification {
 
 extension ZMUserSession {
 
-    public func classification(with users: [UserType]) -> SecurityClassification {
+    public func classification(with users: [UserType], conversationDomain: String? = nil) -> SecurityClassification {
         guard isSelfClassified else { return .none }
 
+        if let conversationDomain = conversationDomain,
+           classifiedDomainsFeature.config.domains.contains(conversationDomain) == false {
```

Comment:
Could be bit shorter with negation operator `!classifiedDomainsFeature.config.domains.contains(conversationDomain)`

Reply:
i think it's easier to read first time when it's `== flase` rather than !